app/assets/stylesheets/articles.scss

```diff
@@ -4,297 +4,74 @@
 @import 'variables';
 @import '_mixins';
 
+
 .home {
   position: relative;
-  min-height: 440px;
-  margin: auto;
+  min-height: 440px; // TODO: why?
+  margin: 0 auto;
   max-width: 1250px;
   text-align: left;
-  @media screen and (min-width: 950px) {
-    margin-top: 26px;
-  }
-  &.sub-home {
-    margin-top: 10px;
-    .side-bar {
-      @media screen and (min-width: 950px) {
-        margin-top: 12px;
-      }
-    }
-  }
-  .query-results-nothing {
-    padding: 40px 0px;
-    font-size: 1.2em;
-    @include themeable(background, theme-container-background, white);
-    @include themeable(box-shadow, theme-container-box-shadow, $bold-shadow);
-    border-radius: 3px;
-    @include themeable(
-      border,
-      theme-container-border,
-      1px solid $outline-color
-    );
-    width: 94%;
-    margin: auto;
-    margin-top: 12px;
-    @media screen and (min-width: 950px) {
-      width: 100%;
-    }
-  }
 
   .articles-list {
-    text-align: left;
     margin: auto;
-    text-align: center;
-    width: 100%;
+    max-width: 100%;
     position: relative;
     -webkit-tap-highlight-color: rgba(0, 0, 0, 0); //Invisible black
-    max-width: 750px;
-    padding-bottom: 30px;
-    @media screen and (min-width: 750px) {
-      margin-bottom: 30px;
-    }
+    box-sizing: border-box;
```

Comment:
Just a general comment about `box-sizing`. We should probably have everything default to this as it simplifies calculating widths/heights.

Reply:
hahah, I [left a comment](https://github.com/thepracticaldev/dev.to/blob/b8dcb0176d3cf2872ca4e2c17b232af61dbb3e1f/app/assets/stylesheets/articles.scss#L215) in that very file to myself that this should be a global thing. 

So yea, 100% agree - but again: I didn't want to refactor everything in that PR. This change would actually cause a lot of harm right away :)